samples/system-test/inspect.test.js

```diff
@@ -69,6 +69,34 @@ it('should report string inspection handling errors', async () => {
   assert.strictEqual(new RegExp(/Error in inspectString/).test(output), true);
 });
 
+// inspect_table
+it('should inspect a table', async () => {
+  const output = await tools.runAsync(
+    `${cmd} table "col1,col2" "Alice,no email address" "Gary,gary@example.com"`,
+    cwd
+  );
+  assert.strictEqual(new RegExp(/Info type: EMAIL_ADDRESS/).test(output), true);
```

Comment:
Same feedback as the other PR - it would be really nice to use the `assert.matches` assertion from `chai` for debugability :)

Reply:
Done - only used for the new tests so as not to duplicate changes from the other PR. Kept native `assert` around for `assert.rejects`.